site/content/docs/user/known-issues.md

```diff
@@ -150,7 +150,7 @@ Makefile:92: recipe for target 'all' failed
 !!! [0116 08:30:54]  1: build/../build/common.sh:518 kube::build::run_build_command_ex(...)
 !!! [0116 08:30:54]  2: build/release-images.sh:38 kube::build::run_build_command(...)
 make: *** [quick-release-images] Error 1
-ERRO[08:30:54] Failed to build Kubernetes: failed to build images: exit status 2
+ERROR[08:30:54] Failed to build Kubernetes: failed to build images: exit status 2
```

Comment:
good catch. :smile: 

Thanks.

Reply:
this is how logrus outputs the errors.
we should not change ERRO->ERROR

Reply:
Ok, I will go to restore it.

Reply:
> this is how logrus outputs the errors.
> we should not change ERRO->ERROR

@neolit123  My mistake! Before I commented, I did the following test. 
But I mistaken our own `Error` as the `ERRO` here :worried: 

```
(MoeLove) ➜  /tmp docker run --rm -it docker
/ # wget -O kind https://github.com/kubernetes-sigs/kind/releases/download/v0.5.1/kind-linux-amd64
Connecting to github.com (52.74.223.119:443)
Connecting to github-production-release-asset-2e65be.s3.amazonaws.com (52.216.112.187:443)
kind                 100% |*****************************************************************| 39.1M  0:00:00 ETA
/ # chmod +x kind
/ # ./kind --loglevel debug get clusters
DEBU[15:12:00] Running: /usr/local/bin/docker [docker ps -q -a --no-trunc --filter label=io.k8s.sigs.kind.cluster --format {{.Names}}\t{{.Label "io.k8s.sigs.kind.cluster"}}] 
Error: could not list clusters: failed to list nodes: exit status 1
```

Reply:
no worries.